src/app/services/ckan.service.ts

```diff
@@ -2,70 +2,96 @@ import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable, catchError, map, of, from, mergeMap, toArray } from 'rxjs';
 import { environment } from 'src/environment/environment';
-import { Dataset } from '../interfaces/dataset-details.interface';
+import { Dataset } from '../interfaces/dataset-details';
+import { CatalogueDetail } from '../interfaces/catalogue-details';
+import { PartialDataset } from '../interfaces/dataset';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class CkanService {
-  ckanToDcat: Map<string, string> = new Map(Object.entries({ package: "Dataset", organization: "Catalogue", tag: "Keyword", group: "Theme" }));
+  private static readonly ckanToDcat: Map<string, string> = new Map(
```

Comment:
Can you please rename this variable to upper snake case, as it is static and final?